aqt/settings.ini

```diff
@@ -17,7 +17,7 @@ retry_backoff: 0.1
 max_retries_on_checksum_error: 5
 max_retries_to_retrieve_hash: 5
 hash_algorithm: sha256
-INSECURE_NOT_FOR_PRODUCTION_ignore_hash: False
+INSECURE_NOT_FOR_PRODUCTION_ignore_hash: True
```

Comment:
@Kidev You accidentally changed this in 7a2a41b66d9cdecacdcd812e738b4117e1356dc5. Also maybe worth cleaning up the commits from earlier so it doesn't clutter the history. Let me show you something, maybe it's obvious, but I didn't know GitHub Desktop had a "Reorder Commit" feature until just recently. I find it quite powerful because it's kind of an alternate way to interactive rebase, when used in combination with squash. To be honest I've never figured out how to do a real interactive rebase, I used to get so annoyed with it I would just hard reset back to some point and then cherry-pick back in and amend stuff as I go 🤣. I was amazed when I figured out the reorder/squash trick.

https://github.com/user-attachments/assets/d29d4f7d-7aa6-4257-96c2-550609a34c3e

Reply:
I usually don't squash in PR because I either squash merge, so it's later and the branch keeps all history, or I merge without rebase, so the history is not cluttered either. I usually reset when I want to trash the latest commits quickly and I'm alone on the branch

I do everything using the CLI I'm too used to it. `git rebase -i` is much easier than it looks:
![output](https://github.com/user-attachments/assets/54573bdf-7778-413e-8ef1-32f0afc0e5e8)

You even have all the commands to use when you enter the interactive mode:
```console
# Commands:
# p, pick <commit> = use commit
# r, reword <commit> = use commit, but edit the commit message
# e, edit <commit> = use commit, but stop for amending
# s, squash <commit> = use commit, but meld into previous commit
# f, fixup [-C | -c] <commit> = like "squash" but keep only the previous
#                    commit's log message, unless -C is used, in which case
#                    keep only this commit's message; -c is same as -C but
#                    opens the editor
# x, exec <command> = run command (the rest of the line) using shell
# b, break = stop here (continue rebase later with 'git rebase --continue')
# d, drop <commit> = remove commit
# l, label <label> = label current HEAD with a name
# t, reset <label> = reset HEAD to a label
# m, merge [-C <commit> | -c <commit>] <label> [# <oneline>]
#         create a merge commit using the original merge commit's
#         message (or the oneline, if no original merge commit was
#         specified); use -c <commit> to reword the commit message
# u, update-ref <ref> = track a placeholder for the <ref> to be updated
#                       to this position in the new commits. The <ref> is
#                       updated at the end of the rebase
#
# These lines can be re-ordered; they are executed from top to bottom.
#
# If you remove a line here THAT COMMIT WILL BE LOST.
#
# However, if you remove everything, the rebase will be aborted.
#```